packages/client/jest.config.js

```diff
@@ -1,11 +1,8 @@
 module.exports = {
-  preset: 'ts-jest',
-  globals: {
-    'ts-jest': {
-      tsconfig: 'tsconfig.json',
-      isolatedModules: true,
-    },
+  transform: {
+    '^.+\\.(m?j|t)s$': '@swc/jest',
   },
+  transformIgnorePatterns: ['@prisma', '\\.prisma'],
```

Comment:
We also don't want to re-transpile `.prisma` folders since we know they are valid cjs.